src/fastcat_devices/fts.cc

```diff
@@ -100,11 +100,17 @@ bool fastcat::Fts::ConfigFromYaml(YAML::Node node)
 
 bool fastcat::Fts::Read()
 {
-  for (int ii = 0; ii < FC_FTS_N_DIMS; ii++) {
-    if (!UpdateSignal(signals_[ii])) {
+  int i= 0;
+  for (auto& signal : signals_) {
```

Comment:
Personally in c++ when I need both an index and the iterate-able variable value, I prefer to loop over the index so I don't forget to increment it (unless I'm dealing with a non-indexable map for example). For my own improvement, is there any advantage do this approach over personal preference?

No change needed

Reply:
Well, one case as you are probably aware is when dealing with member fields, it's nice to use a const iterator as it may perform better.
https://stackoverflow.com/questions/755347/are-const-iterators-faster

Though for this there isn't really a difference because UpdateSignal requires modifying the signal. 
The addition of the integer was merely used for debugging purposes and my preference is using a foreach loop if the index is not useful.